parachain/pallets/verifier-lightclient/src/lib.rs

```diff
@@ -412,20 +408,20 @@ impl<T: Config> Module<T> {
 		new_pruning_range
 	}
 
-	fn verify_receipt_inclusion(block_hash: &H256, proof: &[Vec<u8>]) -> Result<Receipt, DispatchError> {
-		let header = Headers::<T>::get(block_hash)
+	fn verify_receipt_inclusion(proof: &Proof) -> Result<Receipt, DispatchError> {
+		let header = Headers::<T>::get(proof.block_hash)
 			.ok_or(Error::<T>::MissingHeader)?
 			.header;
-	
-		let receipt = header.check_receipt_proof(&proof)
+
+		let receipt = header.check_receipt_proof(&proof.merkle_proof.1)
 			.ok_or(Error::<T>::InvalidProof)?;
-		
+
```

Comment:
Do you have something in your vscode config that removes white space in blank lines?

Reply:
yup! these settings automagically clean up white space on save:

```json
{
  "files.trimFinalNewlines": true,
  "files.trimTrailingWhitespace": true,
  "files.insertFinalNewline": true
}
```